compute/elastic-metal/how-to/use-private-networks.mdx

```diff
@@ -123,6 +123,50 @@ The following commands show how to configure the network interface on an Elastic
   sudo ip addr add 10.10.10.10/24 dev eno1.1234
   ```
 
+## How to configure the Private Network on Windows Server 2019
+
+1. Log into your server as `Administrateur` using the Remote Desktop client.
+
+  <Message type="note">
+
+  The default language of Windows Server installations is French. Make sure to use the user `Administrateur` when connecting to your server. If required you can [change the language of the operating system](/dedibox/dedicated-servers/how-to/change-win-server-language).
+
+  </Message>
+
+2. Launch the **Server Manager** application.
+
+3. Click **Local Server** in the menu on the left. Then click **NIC Teaming** in the **Properties** section.
+
+4. Click **Tasks** > **New Team**. Enter a team name of your choice (e.g. `default`) and tick the checkbox **NIC1**. Then click **OK**.
+
+5. Select the **Team interfaces** tab in the **Adapters and Interfaces** section. Click **Tasks** > **Add interface**. A pop-up displays.
+
+6. Enter a name for the new interface (e.g. `default - VLAN 1234` if your VLAN ID is 1234). Select **Specific VLAN** and enter the ID of your VLAN (e.g. 1234). Click **OK** to confirm.
+
+  <Message type="important">
+
+  Two network interfaces display: Your VLAN and `default`. Make sure to keep the default network as otherwise you lose access to your server via the public Internet.
```

Comment:
```suggestion
  Two network interfaces display: Your VLAN and `default`. Make sure to keep the default network as you will lose access to your server via the public Internet otherwise.
```